### PR TITLE
Feature/cvsb 4817

### PR DESCRIPTION
**Pull request by bsiddappa**
Updated the test type taxonomy with iteration 2 changes.
Removed ADR C&U test types as mentioned in the ticket comments.
Also not considered T tractors in this story as mentioned in the ticket.

### CODE REVIEW

File: tests/resources/test-types.json [hunk 1]
```diff
@@ -1596,42 +1798,7 @@
         ]
       },
       {
-        "id": "53",
-        "name": "ADR C&U",
```
Comment (KHodderDVSA) [inline tests/resources/test-types.json]: Is this no longer needed? Can't see anything in the ticket relating to ADR C&U either way

File: tests/resources/test-types.json [hunk 2]
```diff
@@ -1996,6 +2368,7 @@
             "forVehicleSize": null,
             "forVehicleConfiguration": null,
             "forVehicleAxles": [
+              1,
```
Comment (KHodderDVSA) [inline tests/resources/test-types.json]: Should have trailer entry in "forVehicleType"

File: tests/resources/test-types.json (hunk 1 above)
Comment (KHodderDVSA) [reply tests/resources/test-types.json]: Found the comment on the story which covers this. Resovling

File: tests/resources/test-types.json (hunk 2 above)
Comment (bsiddappa) [reply tests/resources/test-types.json]: Done.

**Pull request closed without merging**